x-pack/elastic-agent/pkg/agent/program/testdata/endpoint_basic-endpoint.yml

```diff
@@ -0,0 +1,86 @@
+fleet:
+  kibana_url: https://kibana.mydomain.com:5601
+  ca_hash: 7HIpactkIAq2Y49orFOOQKurWxmmSFZhBCoQYcRhJ3Y=
+  checkin_interval: 5m
+
+output:
+  elasticsearch:
+    hosts: [127.0.0.1:9200, 127.0.0.1:9300]
+    username: elastic
+    password: changeme
+    api_key: TiNAGG4BaaMdaH1tRfuU:KnR6yE41RrSowb0kQ0HWoA
+    ca_sha256: 7HIpactkIAq2Y49orFOOQKurWxmmSFZhBCoQYcRhJ3Y=
+
+type: endpoint
```

Comment:
Is this correct or should it have `inputs: ` + array?

Reply:
This has been updated to the newest config.